AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3819,7 +3822,79 @@ function Debug-AzStorageAccountEntraKerbAuth {
                 Write-Error $_
             }
         }
-
+        #
+        # AAD Object check 
+        #
+        if (!$filterIsPresent -or $Filter -match "CheckAADObject")
+        {
+            try {
+                $checksExecuted += 1;
+                Write-Verbose "CheckADObject - START"
+                $Context = Get-AzContext
+                $TenantId = $Context.Tenant
+                Connect-MgGraph -Scopes "Application.Read.All" -TenantId $TenantId

```

Comment:
At some point down the line, we will need to port this to work in multiple cloud environments. See: https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.authentication/connect-mggraph?view=graph-powershell-1.0#example-11-connecting-to-an-environment-or-cloud